Dockerfile

```diff
@@ -23,4 +23,6 @@ COPY --from=builder --chown=nobody /etc/ssl/certs/ca-certificates.crt /etc/ssl/c
 COPY --chown=nobody ./config.yaml .
 COPY --from=builder --chown=nobody /go/src/github.com/mendersoftware/deviceconnect/deviceconnect /usr/bin/
 
+STOPSIGNAL SIGUSR1
+
```

Comment:
I would keep the old `SIGTERM` stop signal since this graceful shutdown is a special shutdown condition that should be target by lifecycle hooks.

Reply:
Also, I believe this change might cause all integration tests to run for 1m longer.